sklearn/linear_model/_huber.py

```diff
@@ -37,7 +37,7 @@ def _huber_loss_and_gradient(w, X, y, epsilon, alpha, sample_weight=None):
     alpha : float
         Regularization parameter.
 
-    sample_weight : ndarray, shape (n_samples,), optional
+    sample_weight : ndarray, shape (n_samples,), default=None
```

Comment:
```suggestion
    sample_weight : ndarray of shape (n_samples,), default=None
```

There are some similar occurrences above for `w`, `X` and `y` where `, shape` can be replaced by `of shape`.